### PR TITLE
Adding past eda coding files

### PR DESCRIPTION
**Pull request by bluewhalepig**

### CODE REVIEW

File: finding_CommonID.jl [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+### Finding matches to "cleared" recovered Malawi samples (Guiliherme) 
+
+#Load the csv with all the Malawi samples that we have in lab, "tenative" samples. 
+tentative_samples_list = CSV.read("", DataFrame) #DataFrame is the type that it produces
+
+#Load the csvs with all the recovered Malawi samples that are 'cleared' 
+ID1 = dropmissing(CSV.read("KhulaMW_BiospecimenTemplate_T1_10.03.2024.xlsx")) #dropmissing 
+ID2 = dropmissing(CSV.read("KhulaMW_BiospecimenTemplate_T2_10.03.2024.xlsx"))
+ID3 = dropmissing(CSV.read("KhulaMW_BiospecimenTemplate_T3_10.03.2024.xlsx"))
```
Comment (Hugemiler) [inline finding_CommonID.jl:11]: Ideally, as the project gets more mature, we will transition from those filenames to something different. Usually, we do not share Excel or CSV files containing subject metadata on github. Let's talk about that in the future =]

File: histograms_EDA.jl [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+#Plot a histogram of cogScores 
+cogScores = recovered_data[:, "cogScore"]
+hist(bar_labels= "Counts of CogScores", cogScores, bins = 15, normalization = :none)
+
+#Plot a scatterplot of cogScores vs age 
+scatter(ages, cogScores, title ="Cognitive Scores vs Age", xlabel="Age (months)", ylabel="Cognitive Scores")
+
+
+#build blank figure 
+#Axis object
```
Comment (Hugemiler) [inline histograms_EDA.jl:23]: Is this leftover from code that you deleted?

File: pca_EDA.jl [hunk 3]
```diff
@@ -0,0 +1,75 @@
+### My attempt at PCA in julia! 
+
+using MultivariateStats
+using CairoMakie
+#using Plots
+#using PlotlyJS
```
Comment (Hugemiler) [inline pca_EDA.jl:6]: It's safe to delete those two rows now

File: pca_EDA.jl [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+fig = Figure()
+
+#Create axis 
+ax = Axis(fig[1,1])
+
+#Modifies the axis to contain data 
+sc_age = scatter!(ax, model.U[:,1], model.U[:,2], color = data.ageMonths, alpha=0.5)
+
+#Coloring plot by age
+scatter(model.U[:,1], model.U[:,2], color = data.ageMonths, alpha=0.5)
```
Comment (Hugemiler) [inline pca_EDA.jl:58]: No reason to have this line, now that you created the plot on 49-55, right?

File: pca_EDA.jl [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+
+#using PlotlyJS --> Layout(title="relationship between samples based on taxa abundances and colored by child age in months", yaxis_title="MDS2", xaxis_title="MDS1")
+title!("relationship between samples based on taxa abundances and colored by child age in months")
+xlabel!("MDS1")
+ylabel!("MDS2")
```
Comment (Hugemiler) [inline pca_EDA.jl:63]: It is ok to remove the PlotlyJS logic now that we have settled for Makie.jl. Also, let's practice using the annotation arguments to the `Axis()` function to customize the axis elements =]

File: sample_data.csv
Comment (Hugemiler) [inline sample_data.csv:1]: We should avoid commiting data to github for now, especially at such a high volume. It is usually not done, and often a bad practice. Also, for people like us that so often work with unpublished human metadata and community profiles, it's always a good idea to basically add all `*.csv` files to `.gitignore` just to make sure it doesnt get shared by mistake. Please delete this CSV file from the commit (no harm done this time, this data specifically is already publicly available via other sources). We will discuss how to share data in the future.

**Review by Hugemiler — CHANGES_REQUESTED** [review]
Great job with your first Pull Request! Now you should read the comments, work on resolving them and commit the changes you make.

File: finding_CommonID.jl (hunk 1 above)
Comment (bluewhalepig) [reply finding_CommonID.jl:11]: 👍

File: sample_data.csv
Comment (bluewhalepig) [reply sample_data.csv:1]: I seeee 👍

**Pull request merged**